packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -502,6 +512,11 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
 
     const containerProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(buttonProps, [], ['disabled']);
 
+    // Add additional props to apply on primary action button
+    if (primaryActionButtonProps) {
+      assign(buttonProps, { ...primaryActionButtonProps });
```

Comment:
Is there a reason for this spread? I'm asking due to a TS3.7 upgrade that's exposing some `assign` bugs we have in our codebase. Could this not just be:

```tsx
assign(buttonProps, primaryActionButtonProps);
```

Reply:
@JasonGore it probably could! I think at the moment I didn't realize this, I don't think it should cause any problems.